django/db/models/functions/math.py

```diff
@@ -0,0 +1,117 @@
+from django.db.models import Func, Transform, fields
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class Acos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = fields.FloatField()
+
+
+class Asin(Transform):
+    function = 'ASIN'
+    lookup_name = 'asin'
+    output_field = fields.FloatField()
+
+
+class Atan(Transform):
+    function = 'ATAN'
+    lookup_name = 'atan'
+    output_field = fields.FloatField()
+
+
+class Atan2(Func):
+    function = 'ATAN2'
+    arity = 2
+    output_field = fields.FloatField()
+
+
+class Ceiling(Transform):
```

Comment:
In [#28643 (comment:9)](https://code.djangoproject.com/ticket/28643#comment:9) it recommends naming this function `Ceil` which feels more Pythonic as we have `math.ceil()`.

Please rename this function to `Ceil` and change `lookup_name` to `ceil`.